hypershift-operator/controlplaneoperator-overrides/assets/overrides.yaml

```
@@ -1,4 +1,6 @@
-overrides:
+patforms:
```

isn't it better to make the different platforms separate fields. so we one can't have multiple entries in the list with the same platform.
```suggestion
paltforms:
 aws:
   overrides:
    - version: x.y.z
    .....
```

---

platforms is still spelled wrong in the suggestion 😅

---

👍 makes sense